frontend/components/proposalView.tsx

```diff
@@ -0,0 +1,7 @@
+interface IProposalViewProps {}
+
+const ProposalView: React.FunctionComponent<IProposalViewProps> = (props) => {
```

Comment:
what's this component for?

Reply:
I was initially going to separate the modal from the proposal so i created this — but forgot to delete. Not currently being used!